### PR TITLE
fix(*): Remove deprecated properties #7053

### LINKED ISSUE #7053: Remove deprecated options for 9.1.0
## Description  
Find deprecated properties (@DeprecateProperty) and remove them. This is valid only for deprecated properties for versions <= 9.0.x . Do not remove newly added deprecated decorators for 9.1.x.

### PR DESCRIPTION
**Pull request by mddragnev**
Closes #7053   

### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [ ] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code
 - [ ] This PR includes API docs for newly added methods/properties
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/directives/text-highlight/text-highlight.directive.ts [hunk 1]
```diff
@@ -35,10 +35,6 @@ export interface IActiveHighlightInfo {
      * The column index of the highlight. This property is deprecated, use `column` instead.
      */
     columnIndex?: number;
-    /**
-     * The page index of the highlight. This property is deprecated.
-     */
-    page?: number;
```
Comment (DiyanDimitrov) [inline projects/igniteui-angular/src/lib/directives/text-highlight/text-highlight.directive.ts:41]: You can remove rowIndex and columnIndex properties as well.

File: projects/igniteui-angular/src/lib/directives/text-highlight/text-highlight.directive.ts [hunk 2]
```diff
@@ -328,14 +312,14 @@ export class IgxTextHighlightDirective implements AfterViewInit, AfterViewChecke
...
     /**
-     * Activates the highlight if it is on the currently active row, column and page.
+     * Activates the highlight if it is on the currently active row and column.
      */
     public activateIfNecessary(): void {
         const group = IgxTextHighlightDirective.highlightGroupsMap.get(this.groupName);
-        const column = group.columnIndex === undefined ? group.column : group.columnIndex;
-        const row = group.rowIndex === undefined ? group.row : group.rowIndex;
+        const column = group.column === undefined ? group.column : group.column;
+        const row = group.row === undefined ? group.row : group.row;
```
Comment (DiyanDimitrov) [inline projects/igniteui-angular/src/lib/directives/text-highlight/text-highlight.directive.ts:320]: These conditions doesn't make sense after the rowIndex/columnIndex removal.

Comment (kdinev) [reply projects/igniteui-angular/src/lib/directives/text-highlight/text-highlight.directive.ts:320]: Yeah, I guess remove the declarations and just use `group.column` and `group.row`

Comment (ChronosSF) [reply projects/igniteui-angular/src/lib/directives/text-highlight/text-highlight.directive.ts:320]: Shame on me. I'll fix it :)

**Pull request merged**